about.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Maenza Portfolio - About Page</title>
+<meta name="description" content="Resume for Martin Maenza" />
+<meta name="keywords" content="" />
+<meta http-equiv="author" content="Martin C. Maenza" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="stylesheet" type="text/css" href="styles/style0.css">
+</head>
+
+<body>
+<div class="wrapper">
+<header>
+<!-- page header with logo and navigation controls -->
+   <div class="logo">
+      <a href="index.html"><img src="images/MMportfolio.jpg" alt="Martin Maenza's Portfolio logo" /></a>
+   </div>
+   <nav>
+      <a href="about.html"><img src="images/about-button.jpg" alt="about button" /></a><br />
```

Comment:
Images are generally no longer used as navigational items. Instead, use text and style the button to look like the image with CSS. Every image is a request that gets made to the server and it gets to be expensive in terms of performance/speed.